operatorcontrolplane/v1alpha1/types_conditioncheck.go

```diff
@@ -110,6 +110,12 @@ type OutageEntry struct {
 	// +optional
 	// +nullable
 	End metav1.Time `json:"end,omitempty"`
+
+	// Logs contains log entries related to this outage. Should contain the original
```

Comment:
I think we're going to want `StartLogs` and `EndLogs`.  So we always have the first few and then always have the last few.  But I like the idea.

Reply:
Done